pom.xml

```diff
@@ -75,7 +75,7 @@
         <dependency>
             <groupId>com.exasol</groupId>
             <artifactId>exasol-testcontainers</artifactId>
-            <version>2.0.0</version>
+            <version>2.0.1</version>
```

Comment:
It's a new feature, isn't it? I would change the version to 2.1.0

Reply:
Right idea, wrong entry. I raised the content of the tag `<version>` directly under `<project>` to 0.2.0.